ss_rules.json

```diff
@@ -0,0 +1,792 @@
+			"multiple":	"True",
+			"styles": [
+				"Halftitle Book Title (htit)"
+			],
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Halftitle (sht)"
+				]
+			}
+		}
+	},
+	"Section-Titlepage (sti)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
+			"styles": [
+				"Titlepage Author Name (au)",
+				"Titlepage Book Subtitle (stit)",
+				"Titlepage Book Title (tit)",
+				"Titlepage Cities (cit)",
+				"Titlepage Contributor Name (con)",
+				"Titlepage Imprint Line (imp)",
+				"Titlepage Logo (logo)",
+				"Titlepage Publisher Name (pub)",
+				"Titlepage Reading Line (rl)",
+				"Titlepage Series Title (ser)",
+				"Titlepage Translator Name (tran)"
+			],
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Titlepage (sti)"
+				]
+			}
+		}
+	},
+	"Section-Copyright-Page (scr)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
+			"styles": [
+				"Copyright Text double space (crtxd)",
+				"Copyright Text single space (crtx)"
+			],
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Copyright-Page (scr)"
+				]
+			}
+		}
+	},
+	"Section-Dedication (sde)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
+			"styles": [
+				"Dedication (ded)",
+				"Dedication Author (dedau)"
+			],
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Dedication (sde)"
+				]
+			}
+		}
+	},
+	"Section-Epigraph (sep)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
+			"styles": [
+				"Epigraph - non-verse (epi)",
+				"Epigraph - verse (epiv)",
+				"Epigraph in Text - non-verse (tepi)",
+				"Epigraph in Text - Source (teps)",
+				"Epigraph in Text - verse (tepiv)",
+				"Epigraph Source (eps)"
+			],
+			"position":	"frontmatter",
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Epigraph (sep)"
+				]
+			}
+		}
+	},
+	"Section-Contents (stc)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
+			"styles": [
+				"TOC Author (cau)",
+				"TOC Backmatter Head (cbmh)",
+				"TOC Chapter Number (ccn)",
+				"TOC Chapter Subtitle (ccst)",
+				"TOC Chapter Title (cct)",
+				"TOC Frontmatter Head (cfmh)",
+				"TOC Level-1 Chapter Head (ch1)",
+				"TOC Page Number (cnum)",
+				"TOC Part Number (cpn)",
+				"TOC Part Title (cpt)"
+			],
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Contents (stc)"
+				]
+			}
+		}
+	},
+	"Section-Introduction-Frontmatter (sif)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
+			"styles": [
+				"FM Head (fmh)",
+				"FM Head ALT (afmh)",
+				"FM Head Nonprinting (fmhnp)",
+				"FM Title (fmt)"
+			],
+			"first_child": {
+				"text": [
+					"Introduction"
+				],
+				"match":	"True"
+			},
+			"previous_sibling": {
```

Comment:
Is this saying that the Frontmatter introduction could be preceded by any and all other types of sections?